src/main/java/io/confluent/connect/elasticsearch/RetryUtil.java

```diff
@@ -104,24 +104,23 @@ public static long computeRetryWaitTimeInMillis(int retryAttempts,
    * @param description      present tense description of the action, used to create the error
    *                         message; may not be null
    * @param function         the function to call; may not be null
-   * @param maxAttempts      maximum number of attempts
+   * @param maxTotalAttempts      maximum number of total attempts, including the first call
```

Comment:
Nit: fix the indentation
```suggestion
   * @param maxTotalAttempts maximum number of total attempts, including the first call
```